### DISCUSSION TITLE
NFT image is not showing in phantom

### DISCUSSION THREAD

**Discussion by johnsonchau-bulb** [phantom/docs#96] [Help & Support] [answered]
### Summary

Collection NFT and normal NFT display image is not rendering on chrome extension and mobile.

Note that this is in devnet. And metadata schema follows metaplex standards and I've read this [guide](https://docs.phantom.app/best-practices/tokens/collectibles-nfts-and-semi-fungibles) about best practice for displaying NFTs in phantom.

See:
- Collection NFT: https://solscan.io/token/AyEGfGTxmKEqp7o1fH2Up7mC2dX3VJAdqBb2UTvLYr4f?cluster=devnet#metadata
- Normal NFT: https://solscan.io/token/4EhCVik9zsTqY1ffcwy8YG7znzu5HTo5ThZhHtsWiQBq?cluster=devnet#metadata 

### Steps to Reproduce

View NFT from collectibles tab in Phantom wallet
<img width="361" alt="Screenshot 2023-09-12 at 3 52 14 pm" src="https://github.com/phantom/docs/assets/93719001/49a71878-d2da-43b8-be95-d087c3e7757e">

Note that the exact same NFTs can be viewed in Solflare.
<img width="374" alt="Screenshot 2023-09-12 at 3 55 20 pm" src="https://github.com/phantom/docs/assets/93719001/fb4f438a-21ca-4f47-80cd-2790e3ff8131">

### Phantom Version

23.12.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hi @johnsonchau-bulb,

Thanks for writing in! I'm looking into this and will follow up as soon as I have more info.

**Reply by adamdelphantom** [reply]
hi @johnsonchau-bulb the NFT has no metadata currently.

**Reply by johnsonchau-bulb** [reply]
@adamdelphantom I recently reminted it. Can you look at this one instead? https://solscan.io/token/DrEMsUnYqycNSAgzNymxYfYn1rXt4ARN5NRWoDY9BpjB?cluster=devnet

**Reply by adamdelphantom** [reply, marked as answer]
I've refreshed the cache on that token. Please check to see if this token is rendering again.
